library/modules/Filesystem.cpp

```diff
@@ -99,7 +99,13 @@ bool Filesystem::mkdir (std::string path)
 {
     int fail;
 #ifdef _WIN32
-    fail = ::_mkdir(path.c_str());
+    ::std::wstring wide_path;
+    int convertResult = MultiByteToWideChar(437, 0, path.c_str(), (int)strlen(path.c_str()), NULL, 0);
+    if (convertResult > 0) {
+        wide_path.resize(convertResult+10);
```

Comment:
Why not `+1`?

Reply:
Mostly due to the source I was copy/pasting from (and lack of sleep).